mantle/kola/harness.go

```diff
@@ -175,6 +175,10 @@ var (
 			desc:  "core dump",
 			match: regexp.MustCompile("[Cc]ore dump"),
 		},
+		{
+			desc:  "systemd ordering cycle",
+			match: regexp.MustCompile("Ordering cycle found"),
```

Comment:
```suggestion
			match: regexp.MustCompile("Found ordering cycle"),
```

Reply:
That's the message in the journal. The message printed on the console (which kola checks) is different. I confirmed this catches https://github.com/coreos/fedora-coreos-tracker/issues/660:

```
$ kola run --qemu-image fedora-coreos-33.20201101.1.0-qemu.x86_64.qcow2 basic
2020-11-04T14:49:06Z kola: Found non-executable file with shebang: install-wrappers.sh
2020-11-04T14:49:06Z kola: Found non-executable file with shebang: itest-bare-unit.sh
=== RUN   basic
=== RUN   basic/ServicesActive
=== RUN   basic/ReadOnly
=== RUN   basic/Useradd
=== RUN   basic/MachineID
=== RUN   basic/DbusPerms
=== RUN   basic/NetworkScripts
=== RUN   basic/FCOSGrowpart
=== RUN   basic/PortSSH
--- FAIL: basic (37.40s)
    --- PASS: basic/ServicesActive (0.14s)
    --- PASS: basic/ReadOnly (0.13s)
    --- PASS: basic/Useradd (0.42s)
    --- PASS: basic/MachineID (0.11s)
    --- PASS: basic/DbusPerms (0.23s)
    --- PASS: basic/NetworkScripts (0.12s)
    --- PASS: basic/FCOSGrowpart (0.15s)
    --- PASS: basic/PortSSH (0.13s)
        harness.go:936: Found systemd ordering cycle on machine a82a4def-e9d7-4725-84bf-2b8644425926 console
FAIL, output in _kola_temp/qemu-unpriv-2020-11-04-0949-186549
Error: harness: test suite failed
2020-11-04T14:49:44Z cli: harness: test suite failed
```